src/scss/06_components/paragraphs/_hero.scss

```diff
@@ -70,6 +70,30 @@ $koro-height: 50px;
   }
 }
 
+[dir='rtl'] .hero__arrow {
+  left: auto;
+  right: $spacing-and-half;
+
+  @include breakpoint($breakpoint-l) {
+    right: $spacing-double;
+  }
+}
+
+// Hero image author styles.
+.hero__image-author {
+  // Arrow element width.
+  margin-left: 100px;
+  margin-top: $spacing;
+  text-align: right;
```

Comment:
You could use: `text-align: end;` so that you do not need to override it for the `[dir="rtl"]`